etc/picongpu/taurus-tud/V100.tpl

```diff
@@ -55,9 +55,10 @@
 # That is, replace in the following line the two appearances of 6 with 3.
 .TBG_gpusPerNode=`if [ $TBG_tasks -gt 6 ] ; then echo 6; else echo $TBG_tasks; fi`
 
-# number of cores to block per GPU - we got 28 cpus per gpu
-#   and we will be accounted 28 CPUs per GPU anyway
-.TBG_coresPerGPU=28
+# number of cores to block per GPU - we got 7 cpus per gpu (44cores/6gpus ~ 7cores)
```

Comment:
I think `7 cpus per gpu` is a little imprecise and would be better as `7 cpu cores per gpu`

Reply:
I agree - that confuses my too all the time. Will state it more precisely.